### DISCUSSION TITLE
Reusing `Request`

### DISCUSSION THREAD

**Discussion by shotamakino** [go-resty/resty#588] [Q&A]
Hi, thank you for this great package!

I am trying to clone a `Request` instance in order to reuse it again with a different set of query parameters (for pagination). I was wondering if there's some kind of support for a `Clone` method similar to https://pkg.go.dev/net/http#Request.Clone?

**Comment by jeevatkm** [comment]
@shotamakino Thanks for your suggestion. #718 has been created.
